tdrs-frontend/src/components/PrivateRoute/PrivateRoute.test.js

```diff
@@ -12,46 +12,37 @@ import PrivateRoute from '.'
 describe('PrivateRoute.js', () => {
   const mockStore = configureStore([thunk])
 
-  it('does not return children when user is not authenticated', () => {
-    const store = mockStore({ auth: { authenticated: false } })
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter>
-          <PrivateRoute path="/very-secret-route">
+  const createWrapper = (storeOptions) =>
```

Comment:
We had a lot of tests that followed this same pattern, I abstracted to creation of the private router to a utility to make this test read easier